easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -184,7 +187,8 @@ def __init__(self, path, extra_options=None, build_specs=None, validate=True, hi
 
         # parse easyconfig file
         self.build_specs = build_specs
-        self.parser = EasyConfigParser(filename=self.path, rawcontent=self.rawtxt)
+        self.parser = EasyConfigParser(filename=self.path, rawcontent=self.rawtxt,
+                                        auto_convert_value_types=auto_convert_value_types)
```

Comment:
one less space here before `auto` (use this space where you were missing one before ;))